versions/1.0.0.md

```diff
@@ -38,7 +42,10 @@ An overlay document contains an ordered list of [Action Objects](#overlayActions
 
 ### Versions
 
-TBD
+The Overlay Specification is versioned using a `major`.`minor`.`patch` versioning scheme. The `major`.`minor` portion of the version string (for example 1.0) SHALL designate the Arazzo feature set. `.patch` versions address errors in, or provide clarifications to, this document, not the feature set. The patch version SHOULD NOT be considered by tooling, making no distinction between 1.0.0 and 1.0.1 for example.
```

Comment:
I understand this is what Arazzo is using, but we're currently in discussions on that project regarding what version bumps really mean.  I'm not sure a `SHOULD NOT` reflects the reality.  For example, in Arazzo right now, we're discussing a change in grammar of simply moving a set of parentheses.  That small clarification might constitute a patch change in the spec, but toolmakers may have to consider it, as they could have misinterpreted the spec the first time around. I understand the need to assuage concerns, but I might just leave it out for now.

Reply:
Honestly: "simply moving a set of parentheses" is a breaking/major change unless the spec allows both placements of parentheses and only calls out which placement is preferred/future-proof.

Adding a feature is a minor change.

Rephrasing text without adding or removing a feature is a patch.

Reply:
I'd be happy to drop or simplify this section, but we'll need to clarify it some time and I thought this was pretty lightweight and close to the way we usually manage specifications. Does anyone have a counter-wording to suggest?